src/contentScripts/injected.ts

```diff
@@ -1,17 +1,16 @@
-/* eslint-disable @typescript-eslint/no-use-before-define */
-
 import { MerkleProof } from "@zk-kit/incremental-merkle-tree";
 import log from "loglevel";
 
-import { MerkleProofArtifacts } from "@src/types";
 import { RPCAction } from "@src/constants";
+import { MerkleProofArtifacts } from "@src/types";
+
 import { IRlnGenerateArgs, ISemaphoreGenerateArgs, RlnProofGenerator, SemaphoreProofGenerator } from "./proof";
 
 export type IRequest = {
   method: string;
-  payload?: any;
+  payload?: unknown;
```

Comment:
Are we sure that the payload is `unknown`?

Reply:
Yes, this is a general method which can take any payload type.